src/jit/simd.h

```diff
@@ -49,4 +49,14 @@ struct SIMDIntrinsicInfo
 
 #endif // FEATURE_SIMD
 
+enum SIMD_Level
```

Comment:
It would be good to add a comment here about why this is distinct from `InstructionSet`.
Also, it should be `SIMDLevel`.
From https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#8.11
I'm personally fine with the naming of the values. The coding convention doc actually specifies all-caps with underscores for the enum values, but also acknowledges that the JIT is quite inconsistent.

Reply:
Also, I would move this to the top of the file, so it's clearer that this is outside of the `#ifdef FEATURE_SIMD`.

Reply:
Got it, will do.